### ISSUE TITLE
Some radio settings no longer translated

### ISSUE THREAD

**Issue by ulfhedlund** [EdgeTX/edgetx#4610]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Companion

### Current Behavior

The Rud, Ele, Thr and Ail text in the Radio settings are no longer translated. It is the same in both the Trainer, Hardware and Calibration tabs.

### Expected Behavior

The text is translated, as in 2.9.3.

### Steps To Reproduce

Set language to anything else than English, open Radio settings, go to the Trainer, Hardware or Calibration tab - and voila!

### Version

Nightly (Please give date/commit below)

### Transmitter

RadioMaster TX16S / TX16SMK2

### Anything else?

Nightly as per Feb 2, 2024

**Comment by pfeerick** [comment]
This is most likley due to #4406 and lupdate needing to be run again. Waiting to see if #4613 fixes it.

**Comment by ulfhedlund** [comment]
Yes, lets hope so. However I have run lupdate and updated all SE translations and that didn't work. Hence my post.

**Comment by ulfhedlund** [comment]
I'm sure it will, once all JSON-work is finished. I just ran lupdate and translated all new items for SE and as of today it is still the same.

**Comment by ulfhedlund** [comment]
Not sure if all planned actions have been taken to fix this issue (the translations) but as far as I can see they are not translated in 2.10 rc1 and the issue remains.

**Comment by 3djc** [comment]
As you can see, this issue is still open, meaning that no solution has been applied just yet. It will be closed once they are
